### ISSUE TITLE
Support for `MinimumLevel.Overide()`

### ISSUE THREAD

**Issue by nblumhardt** [serilog/serilog-settings-configuration#10]
The C# API for Serilog now supports:

```
var log  = new LoggerConfiguration()
  .MinimumLevel.Debug()
  .MinimumLevel.Override("Microsoft", LogEventLevel.Warning) 
  .MinimumLevel.Override("MyApp.Buggy.Thing", LogEventLevel.Verbose)
  // ..
```

Not sure how this should map to the JSON syntax provided by this package, but some kind of override mapping would be valuable.

**Comment by nblumhardt** [comment]
Closed in #14.

**Comment by JanEggers** [comment]
Hi just to let you know.....

i used this code so i can stick to the asp.net core default logging configuration:

            var logger = new LoggerConfiguration();
            var configuration = Configuration.GetSection("Logging:LogLevel");
            foreach (var levelConfiguration in configuration.GetChildren() )
            {
                var level = (LogEventLevel) Enum.Parse(typeof (LogEventLevel), levelConfiguration.Value);
                if (levelConfiguration.Key == "Default")
                {
                    logger = logger.MinimumLevel.Is( level );
                    continue;
                }

                logger = logger.MinimumLevel.Override(levelConfiguration.Key, level);
            }
